### ISSUE TITLE
PsySH does not escape backslash and other glitches.

### ISSUE THREAD

**Issue by beyondlimits** [bobthecow/psysh#351]
Windows 7, PHP 7.1.0, PsySH v0.8.1
Since ```\``` character is not escaped properly, it creates ambiguity whether ```"\n"``` is a new line character or a backslash followed by a ````n````.
```
>>> "\n"
=> "\n"
>>> "\\n"
=> "\n"
```
Other strings containing backslash also exhibit weird behavior:
```
>>> "\<<"
=> "<<"
>>> "\\<<"
=> "<<"
>>> "\\\<<"
=> "<<"
>>> "\\\\<<"
=> "<<"
>>> "\\\\\<<"
=> "\<<"
```

**Comment by bobthecow** [comment]
For the `\\n` issue, it should be using color to disambiguate. Does it show the `\` and `\n` in different colors for you?

Some backslash issues are fixed with #359, but the ones with `\<` (and all sorts of variants around it) persist because that's how symphony console escapes their color tags, and IIRC they have some logic in there to prevent double escaping… I'll keep this issue around and try to make some more progress on a fix for this.

**Comment by beyondlimits** [comment]
> For the `\\n` issue, it should be using color to disambiguate. Does it show the `\` and `\n` in different colors for you?

Indeed. I just had to use better terminal than the default one delivered with Windows.

**Comment by bobthecow** [comment]
I know it's been a minute, but this is fixed in 3c260362236b598662c8958fd924b0c81f84cc1b and will be in the next release :) 

<img width="1549" height="651" alt="Image" src="https://github.com/user-attachments/assets/1b09847f-5819-4b4a-972b-ff64e85c48b1" />
